packages/contracts/contracts/core/DAO.sol

```diff
@@ -109,13 +109,9 @@ contract DAO is
     }
 
     /// @inheritdoc PermissionManager
-    function isPermissionRestrictedForAnyAddr(bytes32 _permissionId)
-        internal
-        view
-        virtual
-        override
-        returns (bool)
-    {
+    function isPermissionRestrictedForAnyAddr(
```

Comment:
I didn't change permission manager.
I guess, someone did not use `prettier` as the default formatter.
I can move this changes out of the PR if you want